javascript/transform.js

```diff
@@ -15,27 +16,30 @@ function outOfChina(lat, lng) {
 	return false;
 }
 
-function transformLat(x, y) {
-	var ret = -100.0 + 2.0*x + 3.0*y + 0.2*y*y + 0.1*x*y + 0.2*Math.sqrt(Math.abs(x));
-	ret += (20.0*Math.sin(6.0*x*Math.PI) + 20.0*Math.sin(2.0*x*Math.PI)) * 2.0 / 3.0;
-	ret += (20.0*Math.sin(y*Math.PI) + 40.0*Math.sin(y/3.0*Math.PI)) * 2.0 / 3.0;
-	ret += (160.0*Math.sin(y/12.0*Math.PI) + 320*Math.sin(y*Math.PI/30.0)) * 2.0 / 3.0;
+function gcjTrigLat(x, y) {
```

Comment:
Yeah, size-based readers like V8 would be more happy to optimize small functions.

Humans have infinitely tall screens anyway.